backend/app/cache.js

```diff
@@ -1,6 +1,8 @@
 /* eslint-env node */
 
 const { getLogger } = require('./utils');
+const redis = require('redis');
+const { promisify } = require('util');
```

Comment:
This is more of a note to myself, but I need to make sure the README includes a note that you have to be on node 8 or above, as noted in `package.json`—I was accidentally running node 6 something locally, and this failed because `promisify` wasn't defined. That's a really weird error for someone new to the project to encounter and debug.